dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/ConnectionManagerMBean.java

```diff
@@ -109,4 +109,40 @@
      *
      */
     public String showPoolContents();
+
+    /**
+     * Displays the JndiName of the connection pool associated with
+     * this Connection Manager as a human readable string.
+     *
+     * @return A non-localized string displaying the JndiName of the connection pool
+     *
+     */
+    public String getJndiName();
+
+    /**
+     * Displays the maximum size of the connection pool associated with
```

Comment:
Returning a Long not a String.  Need an update to the Abstract as well as the @return description.  For this method and the following 3.